.github/workflows/ci.yaml

```diff
@@ -44,3 +44,15 @@ jobs:
         run: make build
       - name: test
         run: make test
+  go-mod-tidy-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version-file: go.mod
+          cache: true
```

Comment:
Cache is enabled by default in setup-go@v4 and does not need to be explicitly set.

Reply:
Ok. There is another `cache:true` and I will remove them together later.